4. User Guides/Editing Messages/README.md

```diff
@@ -53,3 +53,9 @@ Various emoji shortcuts exists to display popular text representations as graphi
 Graphical emojis can be enabled/disabled in [Your Account](../managing-your-account/) settings under the **Messages** section.
 
 When graphical emojis are disabled, emoji keywords and shortcuts will be displayed as is  e.g. `:smile:` or `:)`.
+
+## Sending attachments
+
+You can send attachments by clicking on the paperclip icon on the right of the compose message box, which will open a file browser for you to select a supported document and image file to send. Once you have selected a file to attach, a window will be displayed where you can change the file name and add an optional description before sending or cancelling and discarding the attachment.
+
+Rocket.Chat currently does not have support to send multiple attachments at once.
```

Comment:
Hmm... is this expected?  I don't know enough about other chats on this feature to comment; but popular messaging clients such as Whatsapp doesn't seem to have it.   (are we paying homage to email? 🤔   )